mongo/src/org/immutables/mongo/bson4gson/BsonWriter.java

```diff
@@ -176,9 +177,7 @@
       final BigDecimal decimal = (BigDecimal) value;
       try {
         return value(new Decimal128(decimal));
-      } catch (NumberFormatException|AssertionError ex) {
-        // Decimal128 throws AssertionError instead of NumberFormatException for out of range values
-        // see https://jira.mongodb.org/browse/JAVA-2937
+      } catch (NumberFormatException ex) {
```

Comment:
if they fixed `AssertionError` driver should be updated, or we're already using the fixed version?

Reply:
It is really throwing `NumberFormatException` not `AssertionError`

Reply:
ok, false alarm, sorry